clang/include/clang/Frontend/CASDependencyCollector.h

```diff
@@ -55,4 +57,4 @@ class CASDependencyCollector : public DependencyFileGenerator {
 
 } // namespace clang
 
-#endif // LLVM_CLANG_FRONTEND_CASDEPENDENCYCOLLECTOR_H
\ No newline at end of file
+#endif // LLVM_CLANG_FRONTEND_CASDEPENDENCYCOLLECTOR_H
```

Comment:
Apparently my Vim auto-inserts a EOF newline. I can remove this if you prefer (though I plan to cherry pick this on next + the cas branch, so shouldn't cause any problems down the line).

Reply:
This would be nice to keep (it's an annoyance for some of us).  I believe VSCode misses the trailing newline.